CONTRIBUTING_DOCS.md

```diff
@@ -28,31 +39,34 @@ The documentation for our project is written using Markdown. Use our [guidelines
 1. Fork the [OpenVINO™ Toolkit](https://github.com/openvinotoolkit/openvino) repository.
 2. Create a new branch.
 3. Create a new markdown file in an appropriate folder.
-    > **REQUIRED**: The document title must contain a document label in a form: `{#openvino_docs_<name>}`. For example: `Deep Learning Network Intermediate Representation and Operation Sets in OpenVINO™ {#openvino_docs_MO_DG_IR_and_opsets}`.
+    > **REQUIRED**: The document title must contain a document label in a form: `{#openvino_docs_<name>}`. For example: `Deep Learning Network Intermediate Representation and Operation Sets in OpenVINO™ {#openvino_docs_MO_DG_IR_and_opsets}`. This label is a unique identifier that is used to refer to the document from others.
 4. Add your file to the documentation structure. Open the documentation structure file [docs/doxygen/ie_docs.xml](./docs/doxygen/ie_docs.xml) and add your file path to the appropriate section.
 5. Commit changes to your branch.
-6. Create a pull request.
+6. Create a pull request, choose a reviewer or assign to yourself. 
```

Comment:
External contributors (even internal collaborators at Intel who are not part of our GitHub organization) will not be able to add reviewers or assignments. Maybe include a note so this is understood.